### PR TITLE
Adding dependencies to Phovea plugins

### PR DESCRIPTION
**Pull request by thinkh**
@sgratzl Please review if the steps are correct. Then you can merge. Thanks.

### CODE REVIEW

File: contributors/adding-dependencies.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+8. (Optional, but recommended: Commit the changes to your application / plugin repository)
+9. [Update your setup](#update-setup)
+
+
+### pip dependencies
+
+Follow theses steps only, if you cannot find a suitable Phovea plugin (using the Phovea generator in the step above).
+
+1. Navigate to the server library directory
+2. Open the *.yo-rc.json*
```
Comment (sgratzl) [inline contributors/adding-dependencies.md]: see https://github.com/phovea/generator-phovea/issues/81

`.yo-rc.json` should not be manually touched

File: contributors/adding-dependencies.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+1. Navigate to the workspace directory
+2. Run `yo phovea:workspace` to merge the dependencies of all cloned repos in the workspace directory
+3. Override files if you will be asked to
+4. Run `docker-compose build` for server-side changes
+5. Run `npm install` for client-side changes
+
+
+### Stand-alone setup
+
+1. Run `yo phovea:resolve` to clone dependent plugins
```
Comment (sgratzl) [inline contributors/adding-dependencies.md]: not needed

File: contributors/adding-dependencies.md (hunk 1 above)
Comment (thinkh) [reply contributors/adding-dependencies.md]: Is the alternative then to add the dependency directly to the *requirements.txt*?

Comment (sgratzl) [reply contributors/adding-dependencies.md]: you are also adding the dependency directly to the package.json, so yes

**Comment by thinkh** [comment]
Addressed your changes. Ready to merge.

**Pull request merged**